tests/conftest.py

```diff
@@ -31,7 +31,16 @@
     from collections.abc import Iterator
 
 
-class MyProject(models.Project):
+class Platform(models.CraftBaseModel):
```

Comment:
Not strictly related to this, but do we have a full spec of what the platforms section will look like?